frontend/lib/norent/components/footer.tsx

```diff
@@ -48,15 +49,7 @@ export const NorentFooter: React.FC<{}> = () => (
         <div className="column is-8">
           <div className="content is-size-7">
             <FooterLanguageToggle />
-            <Trans id="norent.legalDisclaimer">
-              <p>
-                Disclaimer: The information in JustFix.nyc does not constitute
-                legal advice and must not be used as a substitute for the advice
-                of a lawyer qualified to give advice on legal issues pertaining
-                to housing. We can help direct you to free legal services if
-                necessary.
-              </p>
-            </Trans>
+            <LegalDisclaimer website="JustFix.nyc" />
```

Comment:
I'm _very_ uncertain that `website` should be `JustFix.nyc` here but since this is just a refactoring, I'm leaving it as-is to maintain the same copy as before.